runners/flink/runner/src/test/java/org/apache/beam/runners/flink/streaming/TestCountingSource.java

```diff
@@ -0,0 +1,256 @@
+/*
```

Comment:
I thought about that but couldn't find a way to get a vanilla `UnboundedCountingSource`. Should I add a static method to create one for testing, with a `VisibleForTesting` annotation?